build/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My shop</title>
+    <script type="module" crossorigin src="/assets/index.dc7ce157.js"></script>
+  </head>
+  <body>
+    <div id="app"></div>
+    
+  </body>
+</html>

```

Comment:
I think we don't need to commit build folder, as it contains bundle that only needs to be uploaded to S3 bucket

Reply:
Got it, thanks!